react.js

```diff
@@ -89,5 +89,11 @@ module.exports = {
 				camelcase: 'off',
 			},
 		},
+
+		{
+			// apply testing library rules only to test files
+			files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
```

Comment:
I don't necessarily have a problem with using this regex as a way to activate for test files, but its out of alignment with the configuration in the `shared.js` config.  We should try to standardize the behaviors here either by making this consistent with that, or by making that consistent with this.

Reply:
That's fair.  

Shared has `['**/test/**', '**/*.test.ts', '**/*.test.tsx']` and it includes a few typescript rules and the `test` directory for shared test config/utils/etc.  If I were to change the shared config, it would make these rules apply to JS tests, which one of our consumers still has (for now).  Should I update the shared config and move the typescript rules into a separate override?